src/HearThis/UI/AndroidSyncDialog.cs

```diff
@@ -203,10 +203,64 @@ public MyMessageArgs(byte[] newData)
 		private void _syncButton_Click(object sender, EventArgs e)
 		{
 			AndroidIpAddress = _ipAddressBox.Text;
+			if (AndroidIpAddress.Contains("?"))
+			{
+				MessageBox.Show(
+					"You need to replace the three question marks in the box to the left with the number shown on the device in order to sync manually",
+					"Sorry");
```

Comment:
Rather than using "Sorry" as the title, I think it would be better to either use "HearThis" or the name of the sync dialog itself. Most people ignore the title anyway, but it is generally useful to give the user a clue as to where the message is coming from. (Admittedly, there is less likely to be any confusing in this situation.)
Also, since these messages are complete sentences, go ahead and add a period at the end.